lib/Braintree/ClientTokenGateway.php

```diff
@@ -28,7 +28,7 @@ public function __construct($gateway)
     /**
      * Generate a client token for client-side authorization
      *
-     * @param Optional $params containing request parameters
+     * @param $params Optional, containing request parameters
```

Comment:
```suggestion
     * @param mixed $params Optional, containing request parameters
```

Reply:
For me no problem, but why mixed ?

Reply:
I just ran into this one too. It should be `array`. As it stands, phpstan is looking for a class called `Braintree\Optional`.